controllers/admin/AdminFeaturesController.php

```diff
@@ -460,7 +460,7 @@ public function initContent()
     public function initProcess()
     {
         // Are we working on feature values?
-        if (Tools::getValue('id_feature_value')
+        if (Tools::getValue('id_feature_value') && !Tools::getValue('id_feature')
```

Comment:
```suggestion
        if ((Tools::getValue('id_feature_value') && !Tools::getValue('id_feature'))
```

Reply:
@PierreRambaud Only for better readability, right?

Reply:
Yes, to be sure there is no misunderstanding